pipelines/fbc-release/README.md

```diff
@@ -18,8 +18,8 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                           | Yes       | main                                                            |
 
 ### Changes in 1.6.0
-- modify the task `publish-index-image` to accept the new parameter `buildTimestamp` used
-  to append to the publishing image tag
+- add new parameter `buildTimestamp` when calling the `publish-index-image` task
+  * this new parameter is used for adding a timestamped tag to the index image
```

Comment:
I didn't check when I suggested the change, but it turns out there is actually a slightly different convention of doing the indentation in this file. So this should be:
```
- add new parameter `buildTimestamp` when calling the `publish-index-image` task
  - this new parameter is used for adding a timestamped tag to the index image
```